### DISCUSSION TITLE
authorization: Scope

### DISCUSSION THREAD

**Discussion by ngouy** [adonisjs/core#4605] [Help] [answered]
I can see in an [old adonisjs RFC](https://github.com/adonisjs/rfcs/issues/26#:~:text=Scopes)  that scopes where that authorization scope was discussed, but I can't manage to find anything in the cods/bouncer library that suggest it's implemented.
Am I missing something? If not implemented, what's the correct approach?

**Comment by aikrom** [comment, marked as answer]
[Query scopes](https://lucid.adonisjs.com/docs/model-query-scopes)

**Reply by ngouy** [reply]
Got it. But nothin build in the actual policies. Like in pundit for example

Thanks for details
